test/project_types/rails/commands/create_test.rb

```diff
@@ -87,18 +87,116 @@ def test_can_create_new_app
         FileUtils.rm_r('test-app')
       end
 
+      def test_can_create_new_app_with_db_flag
+        FileUtils.mkdir_p('test-app')
+        FileUtils.mkdir_p('test-app/config/initializers')
```

Comment:
Not really pertinent to this change, but should we remove / create this dir in the test setup? I'm thinking that if a test fails halfway through we may end up with data in this dir that might break subsequent tests.

Reply:
We could use `Dir.mktmpdir`, the name should be unique for each run.

Reply:
(and if used with a block, it should be automaticallly deleted).

Reply:
That's a very nice solution! I could get behind that, but I'm guessing quite a few tests do things this way.